### ISSUE TITLE
Missing "tslib"  dependency 

### ISSUE THREAD

**Issue by lekev** [doug-martin/nestjs-query#925]
**Describe the bug**
Requesting support to use with pnp and yarn2. Currently I see this issues trough yarn 2 -
```Error: @nestjs-query/query-typeorm tried to access tslib, but it isn't declared in its dependencies; this makes the require call ambiguous and unsound.```

**To Reproduce**
1. in nestjs-query project change yarn version: `yarn set version berry` 
2. rub app `yarn nest start`

**Expected behavior**
No depency errors

**Fix**
add "tslib" as depedency in every packages, and 'reflect-metadata' as well in @nestjs-query/core

**Comment by doug-martin** [comment]
If this is for local development to work on `nestjs-query` we use npm with lerna. You can see the commands [here](https://github.com/doug-martin/nestjs-query/blob/master/package.json#L4)

But the basic work flow is

```
npm install #only needs to be run the first time
npm run bootstrap
npm run build
```

**Comment by lekev** [comment]
Your tsconfig.build.json has `{ compilerOptions: { importHelpers: true,... }, ... }` so the build will compile all exports to somethings like this refer to (https://www.npmjs.com/package/tslib)
```
var tslib_1 = require("tslib");
exports.x = {};
exports.y = tslib_1.__assign({}, exports.x);
```
As you can see it compile `require("tslib")`
On a project that use @nestjs-query and with `node_modules` folder as strategy to node depedencies it will probably work fine, another package will surely have `tslint` package defined in a parent packages.json and inside another folder `node_modules`, 
But with "plug and play" aka pnp https://yarnpkg.com/features/pnp the resolver strategy will not check in other packages , and so throw error.

**Comment by doug-martin** [comment]
I got it, thank you for reporting! (I don't use yarn so I haven't run into this before :))

**Comment by doug-martin** [comment]
Fixed in `v0.24.4`. Thank you for submitting a bug report!
